test/darwin/random01.v3

```diff
@@ -0,0 +1,34 @@
+//@execute = true
+import component Math {
+    def log(x : double) -> double;
+}
+
+def SYS_getentropy = 500;
+def main() -> bool {
+    // Hard to test that a number is random but we can at least ensure the SYS_call gives us 
+    // some randomish numbers back 
+    var len = 1000;
+    var a : Array<u8> = Array<u8>.new(len);
+    var err = Darwin.syscall(SYS_getentropy, (Pointer.atContents(a), len));
+    if (err.0 < 0) return false;
+
+    // In node I never saw entropy less than 7 but also measuring 
```

Comment:
This is probably overdoing it; there is a test in `test/linux` that reads 100 bytes from `/dev/random` and checks that at least one byte is not zero. That has a probability of failing of 2^-800.